src/main/java/me/damonkelley/tictactoe/Game.java

```diff
@@ -2,65 +2,83 @@
 
 public class Game {
 
-    private GameRules rules;
     private State state;
-    private Player player1;
-    private Player player2;
-
-    public Game(Player player1, Player player2) {
-        this.player1 = player1;
-        this.player2 = player2;
-        state = new State(player1.getMarker());
-        rules = new GameRules(state);
+    private Marker initialMarker;
+
+    public Game(Marker marker) {
+        initialMarker = marker;
+        state = new State(marker);
     }
 
-    public void nextMove() {
-        nextTurn().move(this);
+    public Game(Marker marker, int boardSize) {
+        initialMarker = marker;
+        state = new State(new Board(boardSize), marker);
     }
 
-    private Player nextTurn() {
-        if (state.getNextMarker() == player1.getMarker()) {
-            return player1;
-        } else {
-            return player2;
+    private Game(State state) {
+        initialMarker = state.getNextMarker();
+        this.state = state;
+    }
+
+    public Game move(Space space, Marker marker) {
+        if (canMove(space)) {
+            state.move(space, marker);
         }
+        return this;
     }
 
-    public State getState() {
-        return state;
+    private boolean canMove(Space point) {
+        return state.getBoard().get(point) == null;
     }
 
-    public Player getWinner() {
-        Marker winningMarker = rules.determineWinner();
+    public Marker nextTurn() {
+        return state.getNextMarker();
+    }
 
-        if (winningMarker == player1.getMarker()) {
-            return player1;
-        } else if (winningMarker == player2.getMarker()) {
-            return player2;
+    public Marker determineWinner() {
```

Comment:
I feel like there is an opportunity to trim this conditional statement. Maybe there is another response that could be returned instead of null that would prevent you from having to do null checks in other methods?